packages/components/src/components/tag/tag.css

```diff
@@ -17,3 +25,28 @@
   padding: var(--tag-padding, 0.5rem);
   font-size: var(--tag-font-size, 0.8rem);
 }
+
+.tag--pill {
+  border-radius: var(--tag-pill-border-radius, 0.75rem);
```

Comment:
default border-radius should be a little bigger (currently it's not perfectly rounded, run `yarn start` to see preview)